botocore/regions.py

```diff
@@ -401,3 +402,34 @@ def _expand_template(
         return template.format(
             service=service_name, region=endpoint_name, dnsSuffix=dnsSuffix
         )
+
+
+class EndpointResolverBuiltins(str, Enum):
+    # The AWS Region configured for the SDK client
+    AWS_REGION: str = "AWS::Region"
```

Comment:
I know I asked for this in the last review but I'm now realizing I was wrong. The values are all strings because they're the enums, I don't think the bool makes sense. The actual typing would be on the object returned from `compute_endpoint_resolver_builtin_defaults` but since it's just a dict, the comments here are fine. Sorry for the churn.

Reply:
Oops, these annotations looked so right I never thought twice about it. Good catch, removed in e0700d3.